.github/workflows/e2e/validate_deployment/action.yaml

```diff
@@ -0,0 +1,45 @@
+name: Validate Deployment
```

Comment:
Should we name this more specifically, like "Validate Deployment Deploy Versions"?